meta-mender-clearfog/templates/local.conf.append

```diff
@@ -1,12 +1,32 @@
 # Appended fragment from meta-mender-community/meta-mender-clearfog/templates
 
+# Uncomment the lines below depending on the model
+
+# Clearfog - Base
 MACHINE ?= "clearfog-base"
+MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-base.bin"
```

Comment:
We can probably set this to:

     MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-${MACHINE}.bin"

To avoid duplicate entries.

Reply:
Didn't even click to do that, good idea.